ansible/roles/bootstrap/templates/bootkube.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Kubernetes Control Plane Bootstrapping
+Documentation=https://github.com/kubernetes-incubator/bootkube
+
+[Service]
+ExecStart=/tmp/bootkube/bin/linux/bootkube start --asset-dir=/tmp/bootkube/assets/  --experimental-self-hosted-etcd --etcd-server=http://127.0.0.1:12379
```

Comment:
you are correct- the current implementation is only a workaround.

I tried `Type=oneshot` first which would be the cleanest, but unfortunately it's synchronous. Also starting `bootkube` directly via ansible is synchronous. We need it in the background though. Another idea would be to simply remove the service file afterwards